docs/adr/0002-use-adr-tools-with-custom-template.md

```diff
@@ -0,0 +1,56 @@
+# 2. Use adr-tools with custom template
+
+Date: 2019-12-18
+
+## Status
+
+Accepted
+
+## Context
+
+It's handy to use tooling to create ADRs. There are a couple of options available.
+
+## Options
+
+### 1. npryce's adr-tools
+
+These appear to be the original tools for managing tools. See
+[npryce/adr-tools](https://github.com/npryce/adr-tools) on GitHub.
+
+#### Pros
+
+- simple, well understood
+- bash scripts
+- flexible enough for us (can customise homedir, templates)
+- also being used by Apply
+
+#### Cons
+
+- unknown
+
+### 2. Markdown Architectural Decision Records (madr)
+
+These are a branch of npryce's adr-tools. See
+[adr/madr](https://github.com/adr/madr). They seem to have added a whole bunch
+of extra stuff that I'm not sure we need, although I do like some of the ideas
+in their ADR template which I want to nick for our own template.
```

Comment:
Maybe utilise rather then nick

Reply:
Never thought id see the day when @defong would be correcting someone else on their language 😄

Reply:
for posterity I did kinda approve of @misaka nicking something before correcting his behaviour...